docs/pages/api-reference/operators.md

```diff
@@ -104,7 +125,40 @@ The `rename` operator has the following parameters:
 1. `columns: Dict[str, str]` - positional argument, that specifies the mapping of old column names to new column names.
 
 :::info
-Fennel does not permit column name collisions post the rename operation. An error is thrown if the new column names are already present in the dataset.
+Fennel does not permit column name collisions post the rename operation. An error is thrown if the new column names are
+already present in the dataset.
+:::
+
+### Dedup
+
+Fennel allows you to discard duplicate entries from a dataset by using the `dedup` operator. A subset of columns can be
+used for identifying duplicates by specifying the `by` parameter.
+
+The `dedup` operator has the following parameters:
+
+1. `by: List[str]` - optional kwarg that specifies the list of fields to use for identifying duplicates. If not
+   specified, all the columns (except the timestamp column) are used for identifying duplicates.
+
+::info
+The `dedup` operator can only be applied on datasets without keys
+:::
+
+### Explode
+
+Fennel allows you to explode `List` type columns into separate rows by using the `explode` operator.
```

Comment:
I'm assuming this is based on Pandas? If so, maybe add a link to pandas explode operator?

Reply:
will do